### DISCUSSION TITLE
Suggestion for additional control near search box - clear contents for next search

### DISCUSSION THREAD

**Discussion by SJK55** [moneymanagerex/moneymanagerex#4704] [Ideas]
When searching for items using the onscreen search box located under the transactions, it would be handy to have a clear function near it.
Without hitting delete on keyboard or over-typing.
Just a suggestion

![image](https://user-images.githubusercontent.com/14268231/175255075-e67921c5-b2f0-4501-af6e-406f0f104b11.png)

**Comment by tactilis** [comment]
To be honest, this feels to me like UI feature creep for what is a very simple action (i.e. clearing the search field).

To clear a field, no matter how many words it contains:

1. Left-click in the field
2. Press Ctrl+A
3. Press Delete

or

1. Left-click and drag across the field to select all characters
2. Press Delete

or if you don't care to use the keyboard...

1. Left-click and drag across the field to select all characters
2. Right-click and select Delete

or

1. Right-click and select Select All
2. Right-click and select Delete


In an app, every extra feature is extra code, no matter how simple, to maintain.

Every extra button is a bit more visual clutter that draws the eyes away from the important UI features.


(Just my thoughts)
